sdk/core/amqp-common/src/ConnectionContextBase.ts

```diff
@@ -63,6 +63,12 @@ export interface ConnectionContextBase {
    * underlying AMQP connection for the EventHub Client.
    */
   cbsSession: CbsClient;
+  /**
+   * Updates the context to use a new underlying AMQP connection,
+   * a new cbs session, and new locks used when establishing an AMQP
+   * connection and negotiating cbs claims.
```

Comment:
This was a little confusing to read, it took me a few seconds to realize that the last part applies just to the locks.
I would skip the part on locks altogether and assume that they are part and parcel of having a new connection and cbs session

Reply:
Removed mention of locks, thank you!